### DISCUSSION TITLE
Gocd - create ressources without Admin permission

### DISCUSSION THREAD

**Discussion by kouata** [gocd/gocd#11336] [Q&A]
Hi Everyone,
I'm Gocd administrator and I would like to give permissions to some peoples (outside my team) to create ressources like pipeline, materials,  package and conf repositories, etc without giving them Admin permission. the idea is to make them autonomous but without the Admin rights.
Is there a way to do it with Gocd?

**Comment by chadlwilson** [comment]
See https://groups.google.com/g/go-cd/c/RDmRvEQCoUk/m/8tr4AasRAAAJ
